src/tools/ts-generator/main.cpp

```diff
@@ -80,34 +75,15 @@ int main(int argc, char* argv[])
     if (! checkOptions(settings))
         return 1;
 
+    // ============ THERMAL : Getting data for generating time-series =========
     auto study = std::make_shared<Data::Study>(true);
     Data::StudyLoadOptions studyOptions;
-    studyOptions.prepareOutput = true;
-
     if (!study->loadFromFolder(settings.studyFolder, studyOptions))
     {
         logs.error() << "Invalid study given to the generator";
         return 1;
     }
 
-    study->initializeRuntimeInfos();
-    // Force the writing of generated TS into output/YYYYMMDD-HHSSeco/ts-generator/thermal[/mc-0]
-    study->parameters.timeSeriesToArchive |= Antares::Data::timeSeriesThermal;
-
-    try
-    {
-        Antares::Check::checkMinStablePower(true, study->areas);
```

Comment:
`Antares::Check::checkMinStablePower(true, study->areas);` : disappeared.
Indeed, we don't need to check min stable power to generate a thermal TS : min stable power is never used to build a thermal TS.